gradio/components/video.py

```diff
@@ -120,6 +124,7 @@ def __init__(
             min_length: The minimum length of video (in seconds) that the user can pass into the prediction function. If None, there is no minimum length.
             max_length: The maximum length of video (in seconds) that the user can pass into the prediction function. If None, there is no maximum length.
             loop: If True, the video will loop when it reaches the end and continue playing from the beginning.
+            streaming: When used set as an output, takes video chunks yielded from the backend and combines them into one streaming audio output.
```

Comment:
```suggestion
            streaming: When used set as an output, takes video chunks yielded from the backend and combines them into one streaming video output.
```

Reply:
Is there some restriction on what the `type` of the chunks can be when streaming? Would be good to document here and in the guide

Reply:
Added!